File: Clase2303/Calculadora.java
package Clase2303;

public class Calculadora {
    
    public double sumar(double a, double b){
        return a + b;
    }
    public double restar(double a, double b){
        return a - b;
    }
    public double multiplicar(double a, double b){
        return a * b;
    }
    public double dividir(double a, double b){
        if(b == 0){
            System.out.println("No se puede dividir entre 0");
            return -1;
        }else{
            return a / b;
        }
    }

    public double promedio(double a, double b, double c){
        return (a+b+c)/3;
    }

    public double[] raices(double a, double b, double c){
        double dis = b*b - 4 *a*c;
        if(dis<0){
            System.out.println("No hay raices reales");
            return null;
        }else if(dis == 0){
            double[] raices = new double[1];
            raices[0] = -b/2*a;
            return raices;
        }else{
            double[] raices = new double[2];
            raices[0] = (-b + Math.sqrt(dis))/2*a;
            raices[1] = (-b - Math.sqrt(dis))/2*a;
            return raices;
        }
    }
}

File: Clase2303/Calculadora/Calculadora.py
class Calculadora:
    
    def sumar(self, a, b):
        return a + b

    def restar(self, a, b):
        return a - b
    
    def multiplicar(self, a, b):
        return a * b
    
    def dividir(self, a, b):
        if(b == 0):
            return "No se puede dividir entre cero"
        else:
            return a / b
    
    # Método que calcula el promedio de N números
    # Metodo que calcula la potencia de un numero elevado a otro
    
    2^3=2*2*2
    2^(1/2) = 
        
    def promedio(self, a, b , c):
        return (a + b + c) / 3
    
    def raices(self, a, b, c):
        dis = b*b - 4*a*c
        if(dis < 0):
            return "NO TIENE RAICES REALES"
        elif(dis == 0):
            return -b/2*a
        else:
            x1 = (-b+dis**(1/2))/(2*a)
            x2 = (-b-dis**(1/2))/(2*a)
            return x1, x2

calc = Calculadora()
print(calc.sumar(3, 4))
print(calc.restar(5, 6))
print(calc.multiplicar(1, 9))
print(calc.dividir(3, 5))
print(calc.promedio(3, 4, 5))
print(calc.raices(4, 10, 3))